### PR TITLE
[ROX-15873] Change vulnerables to multiarch images

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.3+

Issue: https://issues.redhat.com/browse/ROX-15873

Link to docs preview:
- [Verifying installation of RHACS on Red Hat OpenShift](https://67465--docspreview.netlify.app/openshift-acs/latest/installing/installing_ocp/verify-installation-rhacs-ocp)
-  [Verifying installation of RHACS on other platforms](https://67465--docspreview.netlify.app/openshift-acs/latest/installing/installing_other/verify-installation-rhacs-other)

QE review:
- [x] QE has approved this change.

Additional information:
Cherrypick into `rhacs-docs-4.3`

### CODE REVIEW

**Review by kcarmichael08 — APPROVED** [review]

File: modules/verify-acs-installation-ocp.adoc [hunk 1]
```diff
@@ -45,9 +45,9 @@ $ oc new-project test
 [source,terminal]
 ----
 $ oc run shell --labels=app=shellshock,team=test-team \
-  --image=vulnerables/cve-2014-6271 -n test
+  --image=vulnerables/example-vulnerables-cve-2014-6271 -n test
```
Comment (robbycochran) [inline modules/verify-acs-installation-ocp.adoc:48]: You need to use the full image path here: `quay.io/stackrox-io/docs:example-vulnerables-cve-2017-7494`
`quay.io/stackrox-io/docs:example-vulnerables-cve-2014-6271`

**Comment by SNiemann15** [comment]
/assign @robbycochran 

Please QE approve

**Review by robbycochran — APPROVED** [review]

**Pull request merged**